types/misc.ts

```diff
@@ -0,0 +1,8 @@
+import { IMessageAttachment } from '@rocket.chat/apps-engine/definition/messages';
+import { IBlock } from '@rocket.chat/apps-engine/definition/uikit';
+
+export interface IMessageParam {
+    text?: string;
+    blocks?: Array<IBlock>;
+    attachment?: IMessageAttachment;
+}
```

Comment:
This is not being used anywhere. We have a similar interface created as `IDialogflowMessageData`. We should keep only one of them.